lib/DonorFilter/PluggableDonorFilter.test.js

```diff
@@ -0,0 +1,92 @@
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { render, cleanup } from '@testing-library/react';
+import user from '@testing-library/user-event';
+
+import PluggableDonorFilter from './PluggableDonorFilter';
+
+const mockVendorData = { id: '1', name: 'Amazon' };
+
+jest.mock('../Donors', () => ({
+  ...jest.requireActual('../Donors'),
+  useFetchDonors: jest.fn(() => ({
+    isLoading: false,
+    donors: [{ id: '1', name: 'Amazon' }],
+  })),
+  DonorsLookup: jest.fn(({ children, ...rest }) => {
+    return (
+      <div>
+        {children}
+        <button
+          type="button"
+          id={rest?.name}
+          onClick={() => rest?.onAddDonors([mockVendorData])}
+        >
+          Add donor
+        </button>
+      </div>
+    );
+  }),
+}));
+
+const labelId = 'ui-orders.orderDetails.assignedTo';
+
+const queryClient = new QueryClient();
+
+const wrapper = ({ children }) => (
+  <QueryClientProvider client={queryClient}>
+    {children}
+  </QueryClientProvider>
+);
+
+const renderFilter = (props) => (render(
+  <PluggableDonorFilter
+    id="donor"
+    activeFilters={[]}
+    labelId={labelId}
+    name={mockVendorData.name}
+    onChange={() => {}}
+    {...props}
+  />,
+  { wrapper },
+));
+
+describe('PluggableDonorFilter', () => {
+  afterEach(cleanup);
```

Comment:
`afterEach(cleanup)` it triggered automatically

Reply:
updated